### ISSUE TITLE
Fix random failure of TestSetReplyTimeout 

### ISSUE THREAD

**Issue by ondrej-fabry** [FDio/govpp#47]
I have noticed that the test for setting reply timeout fails occasionally. Have not looked much deeper yet into the code, I suspect some timing issue or perhaps improper test procedure in the test case.

```
--- FAIL: TestSetReplyTimeout (0.02s)
    channel_test.go:199: 
        Unexpected error:
            <*errors.errorString | 0xc0003d4370>: {
                s: "no reply received within the timeout period 1ms",
            }
            no reply received within the timeout period 1ms
        occurred
FAIL
FAIL	go.fd.io/govpp/core	0.102s
```

You can reproduce this by running the tests 100 times:

```
go test -count 100 -failfast -race ./...
```

**Comment by sknat** [comment]
Seeing the same intermittent issue on some patches with the following backtrace 
( or see https://github.com/FDio/govpp/actions/runs/3106809622/jobs/5089740774 )
````
ok  	go.fd.io/govpp/codec	0.034s
time="2022-09-27T10:18:29Z" level=warning msg="Unable to process request" channel=1 error="not connected to VPP, ignoring the request" msg_crc=aa610c27 msg_name=sw_interface_dump seq_num=2
time="2022-09-27T10:18:29Z" level=warning msg="Received reply to an already closed binary API request" seqNum=2
time="2022-09-27T10:18:29Z" level=warning msg="ignoring received reply: &{seqNum:2 msgID:101 data:[0 101 0 2 0 2 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: sw_interface_set_flags_reply)" channel=1 expSeqNum=3
--- FAIL: TestReceiveReplyAfterTimeout (0.10s)
    channel_test.go:379: 
        Unexpected error:
            <*errors.errorString | 0xc000039c70>: {
                s: "no reply received within the timeout period 1ms",
            }
            no reply received within the timeout period 1ms
        occurred
time="2022-09-27T10:18:32Z" level=warning msg="VPP health check probe timed out after 250ms (1. timeout)"
time="2022-09-27T10:18:33Z" level=warning msg="VPP health check probe timed out after 250ms (2. timeout)"
time="2022-09-27T10:18:34Z" level=warning msg="VPP health check probe timed out after 250ms (3. timeout)"
time="2022-09-27T10:18:34Z" level=error msg="VPP does not responding, the health check exceeded threshold for timeouts (>2)"
time="2022-09-27T10:18:34Z" level=warning msg="Received reply to an already closed binary API request" seqNum=0
time="2022-09-27T10:18:34Z" level=warning msg="ignoring received reply: &{seqNum:0 msgID:101 data:[0 101 0 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: control_ping_reply)" channel=1 expSeqNum=1
time="2022-09-27T10:18:35Z" level=warning msg="Received reply to an already closed binary API request" seqNum=1
time="2022-09-27T10:18:35Z" level=warning msg="ignoring received reply: &{seqNum:1 msgID:101 data:[0 101 0 2 0 1 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: control_ping_reply)" channel=1 expSeqNum=2
time="2022-09-27T10:18:36Z" level=warning msg="Received reply to an already closed binary API request" seqNum=65534
time="2022-09-27T10:18:36Z" level=warning msg="ignoring received reply: &{seqNum:65534 msgID:101 data:[0 101 0 2 255 254 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: sw_interface_details)" channel=1 expSeqNum=1
time="2022-09-27T10:18:36Z" level=warning msg="Received reply to an already closed binary API request" seqNum=65535
time="2022-09-27T10:18:36Z" level=warning msg="ignoring received reply: &{seqNum:65535 msgID:101 data:[0 101 0 2 255 255 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: sw_interface_details)" channel=1 expSeqNum=1
time="2022-09-27T10:18:36Z" level=warning msg="Received reply to an already closed binary API request" seqNum=0
time="2022-09-27T10:18:36Z" level=warning msg="ignoring received reply: &{seqNum:0 msgID:101 data:[0 101 0 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: sw_interface_details)" channel=1 expSeqNum=1
time="2022-09-27T10:18:36Z" level=warning msg="Unable to process request" channel=1 error="not connected to VPP, ignoring the request" msg_crc=51077d14 msg_name=control_ping seq_num=2
time="2022-09-27T10:18:36Z" level=warning msg="Received reply to an already closed binary API request" seqNum=1
time="2022-09-27T10:18:36Z" level=warning msg="ignoring received reply: &{seqNum:1 msgID:101 data:[0 101 0 2 0 1 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: control_ping_reply)" channel=1 expSeqNum=2
time="2022-09-27T10:18:37Z" level=warning msg="Received reply to an already closed binary API request" seqNum=1
time="2022-09-27T10:18:37Z" level=warning msg="ignoring received reply: &{seqNum:1 msgID:101 data:[0 101 0 2 0 1 0 0 0 0 0 0 0 0 0 0 0 0] lastReceived:false err:<nil>} (expecting: control_ping_reply)" channel=1 expSeqNum=2
FAIL
FAIL	go.fd.io/govpp/core	11.919s
````

**Comment by ondrej-fabry** [comment]
Partially fixed in #54. However running test 100 times still runs into issues. I will update #45 with fix for this issue.

**Comment by ondrej-fabry** [comment]
Fixed in #57
